src/pybind/mgr/dashboard/frontend/src/app/shared/services/task-manager-message.service.ts

```diff
@@ -139,14 +167,18 @@ export class TaskManagerMessageService {
     (metadata) => {
       return Components[metadata.component] || metadata.component || 'Unknown Task';
     },
+    (metadata) => {
+      return (
+        'Progressing ' + (Components[metadata.component] || metadata.component || 'unknown task')
```

Comment:
For consistency across all task-related messages, I recommend you to use `Executing` instead of `Progressing`.